Makefile

```diff
@@ -7,7 +7,7 @@ SRCS = %.go
 all: $(BINARY)
 
 $(BINARY): deps **/**/*.go **/*.go *.go
-	go build -ldflags "-X main.buildCommit `git describe --long --tags --dirty --always`" ./cmd/ssh-chat
+	CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags "-X main.buildCommit `git describe --long --tags --dirty --always`" ./cmd/ssh-chat
```

Comment:
This will break for anyone who wants to run `make` on their own machine that is not linux.